package.json

```diff
@@ -107,14 +107,15 @@
     "@bazel/typescript": "^2.0.3",
     "@babel/cli": "^7.10.5",
     "@commitlint/cli": "^9.1.2",
+    "@figma/plugin-typings": "^1.16.1",
     "@nguniversal/builders": "^10.0.1",
     "@nguniversal/express-engine": "^10.0.1",
-    "@nrwl/angular": "^10.0.7",
-    "@nrwl/jest": "^10.0.7",
-    "@nrwl/linter": "^10.0.7",
-    "@nrwl/node": "^10.0.7",
-    "@nrwl/nx-plugin": "^10.0.7",
-    "@nrwl/workspace": "^10.0.7",
+    "@nrwl/angular": "^10.0.13",
+    "@nrwl/jest": "^10.0.13",
+    "@nrwl/linter": "^10.0.13",
+    "@nrwl/node": "^10.0.13",
+    "@nrwl/nx-plugin": "^10.0.13",
+    "@nrwl/workspace": "^10.0.13",
```

Comment:
Bumping this since 10.0.7 has a bug where the `.nxignore` doesn't have any effect